code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -327,6 +327,17 @@
 	item_state = "duffle-eng"
 	burn_state = FIRE_PROOF
 
+/obj/item/weapon/storage/backpack/dufflebag/engineering/drone/New()
+	..()
+	var/color = pick("red","yellow","green","blue","pink","orange","cyan","white")
+	new /obj/item/weapon/screwdriver(src)
+	new /obj/item/weapon/wrench(src)
+	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/crowbar(src)
+	new /obj/item/stack/cable_coil(src,30,color)

```

Comment:
Instead of this, just do `new /obj/item/stack/cable_coil/random(src)` and remove the var/color above.